### ISSUE TITLE
Some question about Multilayer measures

### ISSUE THREAD

**Issue by smile0227** [braph-software/BRAPH-2#1394]
I am sorry to bother you.  I have been troubled by the calculation of  Multilayer measures  for a long time and I really appreciate your help.
1）After I selected Pipeline Functional Multiplex Analysis BUD, I clicked Load Group FUN MP XLS，and uploaded the data under ..\BRAPH-2-Matlab-develop\braph2\pipelines\functional multiplex\example data FUN_MP (fMRI)\xls\GroupName1 folder.   I observed that each XLS table of each subject is a matrix of 52 rows and 200 columns. If I understand correctly, is it a time series of 52 time points in 200 brain regions?
2）Afer uploading the data, I clicked Analyzed Group,  but I don't quite understand the meaning of ID, label, repetition, etc. How can I set these values to obtain the measures of multi-layer brain network for each subject?
3）I calculate the brain network connection value and MultiplexclusteringAv. Click Graph Data and find that the brain network connection value of each layer is NAN, and it seems that only one subject's connection result appears. The result seems to be wrong.
4）When I clicked open a line for the measure,  similarly, it seems that no results have been obtained. In addition, when I observe the results, why do I need to select the number of layers? Won't the multi-layer brain network measures calculation take the entire network as a whole?
5）In Braph, could I upload the  the multi-layer network that has been calculated instead of the original sequence to get the meaures
I'm sorry to have so many questions and look forward to your reply.

**Comment by giovannivolpe** [comment]
Thanks for using BRAPH! 
Since most of our team is currently at the OHBM conference, we'll check and answer your questions in a couple of days.

In the meantime, could you confirm which version of BRAPH you are using?
(I assume you are using the version currently in the develop branch, I just want to be sure)

**Comment by smile0227** [comment]
Thank you for your reply. I am using the BRAPH-2-Matlab-develop.
In order for you to better correct my operation problems, I uploaded my parameter setting pictures and results：
1) Afer uploading the data, I clicked _Analyzed Group_ and set the _ID, label_ and other parameters as follows:
![image](https://user-images.githubusercontent.com/71267915/174474589-11eda450-6932-44e0-9cac-495dfffb57c0.png)
2）I click _Graph Data_ and find that the brain network connection value of each layer is NAN. It seems that only one subject's connection result appears. 
![image](https://user-images.githubusercontent.com/71267915/174474676-6325360e-156d-4e54-a43f-df0f8ca4a28f.png)
3)When I clicked _open a line_ for the measure，the results obtained are shown in the figure below.  Similarly, it seems that no results have been obtained. In addition, when I observe the results, why do I need to select the number of layers? Won't the multi-layer brain network measures calculation take the entire network as a whole?
![image](https://user-images.githubusercontent.com/71267915/174474712-ead8d2af-adbe-42a8-aa0c-3a2766c5d312.png)

Enjoy the conference, and look forward to your reply.
Best wishes!
